Content.Shared/Weapons/Ranged/Components/BallisticAmmoSelfRefillerComponent.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Content.Shared.Weapons.Ranged.Systems;
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.Weapons.Ranged.Components;
+
+/// <summary>
+/// This component, analogous to <c>BatterySelfRechargerComponent</c>, will attempt insert ballistic ammunition into
```

Comment:
```suggestion
/// This component, analogous to <see cref="BatterySelfRechargerComponent"/>, will attempt insert ballistic ammunition into
```
I'm not sure what `<c>` does, did you mean to use `<see cref>`?

Reply:
`<c>` makes it "code formatted" and I would've used `see`, but this was way back when that component was in Server, and thus not able to be referenced. I can check if I can reference it now.